src/core/exporter.js

```diff
@@ -86,6 +86,8 @@ function cleanup(cloneDoc) {
 
   insertions.appendChild(metaGenerator);
   head.prepend(insertions);
+  console.log("BEFORE SAVE");
+  debugger;
```

Comment:
Stepping here will jump into pubsubhub... and checking `subscriptions` shows that that all the subscriptions have vanished :(

Reply:
I think for whatever reason, this `pubsubhub` export must be different from the document's one... but I honestly don't know.

Reply:
![image](https://user-images.githubusercontent.com/3396686/53862947-3c8b1f80-402c-11e9-8b9c-5a01a4963037.png)

I can see them alive? 🤔

Reply:
Seems to happen in the test harness. If you run Karma, the test fails (for me at least).